component.json

```diff
@@ -2,17 +2,29 @@
   "title": "Splitter",
   "description": "Splits a message into multiple messages.",
   "actions": {
-    "split":{
+    "split": {
```

Comment:
One part of this issue is to mark this action as depricated.

Reply:
Resolved in 09277a4